src/GeositeFramework/js/Export.js

```diff
@@ -1,12 +1,121 @@
+/*global Backbone, _, $ */
+
 require(['use!Geosite',
+         'esri/map',
+         'esri/geometry/Extent',
+         'esri/layers/ArcGISDynamicMapServiceLayer',
+         'esri/layers/ArcGISTiledMapServiceLayer',
+         'dojo/dom-style',
          'dojo/Deferred',
          'dojo/request',
          'framework/Logger'],
-    function(N,
-             Deferred,
-             request,
-             Logger) {
+    function(N) {
     "use strict";
+        function render(view) {
+            var html = N.app.templates['template-export-window']();
+            view.$el.empty().append(html);
+            if (view.$uiContainer) {
+                view.$uiContainer.show();
+            }
+
+            return view;
+        }
+
+        function createUiContainer(view, paneNumber) {
+            var model = view.model,
+                $uiContainer = $($.trim(N.app.templates['template-export-window']({id: 'export-print-preview'})));
+
+            view.$uiContainer = $uiContainer;
+
+            var pluginCssPath = 'css/app-print.css',
+                printCssClass = 'plugin-print-css',
+                oppositePaneHideCssPath = 'css/print-hide-map' +
+                    (paneNumber === 0 ? 1 : 0) + '.css';
+
+
```

Comment:
There should be a call here to remove any plugin print stylesheets:
Something like: 

```
$('.plugin-print-css').remove();
```